CHANGELOG.md

```diff
@@ -1,40 +1,73 @@
 
-## UNDER DEVELOPMENT
+## Release v3.6.0
 
-The "UNDER DEVELOPMENT" section of the Mattermost Desktop changelog appears
-in the product's `master` branch to note key changes committed to `master`
-and are on their way to the next stable release.
-When a stable release is pushed, "UNDER DEVELOPMENT" heading is removed
-from the final changelog of the release.
+Release date: February 28, 2017
 
-Release date: TBD
+Upgrading to Mattermost server 3.6 or later is recommended, as new features for the desktop app have been added following the release of the team sidebar.
 
 ### Improvements
-- Added a new team button next to the team tabs
-
-#### All Platforms
-- Suppress white screen which is displayed for a moment on startup
-- Support unread indications in team sidebar of Mattermost server 3.6
-- Remove confusing `Ctrl(CMD)+S` shortcut for searching within a team
+ - Added support for unread indicators following the release of team sidebar in Mattermost server 3.6
+ - Removed a confusing `CTRL/CMD+S` shortcut for searching within a Mattermost team
+ - Added support for SAML OneLogin and Google authentication for Enterprise users
+ - Streamlined desktop server management:
+   - "Team Management" changed to "Server Management" following the release of team sidebar in Mattermost server 3.6
+   - Added a "+" icon to the desktop server tab bar to more easily sign into a new Mattermost server
+   - Added an option to sign into another Mattermost server from **File > Sign in to Another Server**
+   - Clicking "Add new server" on the Settings page opens a dialog instead of a new row
+   - Clicking "Remove" next to a server now requires a confirmation to prevent a user from removing the server by accident
+   - Clicking "Edit" next to a server on the Settings page opens a dialog
+   - Clicking on a server on the Settings page opens the corresponding server tab
+ - Simplified desktop app options:
+   - App options now auto-save when changed
+   - Added supporting help text for each option
+   - Removed "Leave app running in menu bar when application window is closed" setting for Mac, which is not applicable for that platform
+   - Removed "Toggle window visibility when clicking on the tray icon" setting for Windows, given the behavior is inconsistent with typical Windows app behavior
+   - Removed "Hide menu bar" setting to avoid users not being able to use the menu bar and the Settings page.
 
 ### Bug Fixes
 
 #### All Platforms
-- Do not place the application window outside of the viewable area anymore, if the display it has been on has been removed
-- Remove remaining badges after logging out of the team
-- Fixed an issue where context menu appears many times after reloading
+- Mattermost window no longer opens on a display screen that has been disconnected
+- Mention badges no longer persist after logging out of a Mattermost server
+- Fixed an issue where selecting a server from the system tray icon while viewing the Settings page didn't work
```

Comment:
I feel that it's an improvement because it was not a bug in v3.5. So it would be something like "Now selecting a server from the system tray icon, "Window" menu bar item, or `CTRL/CMD+{n}` shortcut while viewing the Settings page works".